packages/simcore-sdk/tests/integration/conftest.py

```diff
@@ -87,39 +119,41 @@ async def filemanager_cfg(
 
 @pytest.fixture
 def create_valid_file_uuid(project_id: str, node_uuid: str) -> Callable[[Path], str]:
-    def create(file_path: Path) -> str:
+    def _create(file_path: Path) -> str:
         return np_helpers.file_uuid(file_path, project_id, node_uuid)
 
-    return create
+    return _create
 
 
 @pytest.fixture()
 def default_configuration(
     node_ports_config: None,
     bucket: str,
-    pipeline: Callable[[str], str],
-    task: Callable[..., str],
+    create_pipeline: Callable[[str], str],

```

Comment:
insignificant: same here since the type is given, it's pretty clear it's meant to be used as a callable.

Reply:
yes, but In the case of callable fixtures, I think it does not harm adding the prefix ``factory_*`` or a verb that describes an action (e.g. ``create_*`` or ``run_*``).  For other fixtures we could definitively use nouns.

This is an extended practice (check e.g. standard fixtures in pytest like ``path_factory,``) and when you are deep in the test it is much clear of the intention. 

So, I formally propose this "styling rule" to name our callable fixtures :-)